### PR TITLE
add new blog

### PR DESCRIPTION
**Pull request by bitakshat**
Add new blog, **The 5 best exercises to build muscles**.

### CODE REVIEW

**Comment by pandevim** [comment]
This is not a convention but we usually do not put images in `images/` directory.

File: content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+template: "post"
+slug: "the-5-best-exercises-to-build-muscles"
+
+title: "The 5 Best Exerciese-to-build-muscles"
```
Comment (pandevim) [inline content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md:5]: title has unnecessary hyphens

File: content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+author: "Soumyadeep Dubey"
+
+cover: "./banner.jpg"
+images:
+    - banner:
+        url: "https://unsplash.com/photos/TuzrzArccvc"
+        feature: ./banner.jpg
+        credit:
+            info: "Taken from pexels.com"
+            link: "https://unsplash.com/photos/TuzrzArccvc"
```
Comment (pandevim) [inline content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md:15]: author link will go here not the link of the image

File: content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+template: "post"
+slug: "the-5-best-exercises-to-build-muscles"
+
+title: "The 5 Best Exerciese-to-build-muscles"
+author: "Soumyadeep Dubey"
+
+cover: "./banner.jpg"
```
Comment (pandevim) [inline content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md:8]: banner is not showing on the landing page due wrong path should be `./images/banner.jpg`, but eventually you have to remove this as `images/` directory is not required now.
image

File: content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+images:
```
Comment (pandevim) [inline content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md:9]: all images used in the blog has to have a meta, you only defined the `banner` image. See [this](https://github.com/thedarkale/website4/blob/main/content/posts/aniruddha-pandey/1001-movies-you-must-see-before-you-die/index.md) for example.

**Review by pandevim — CHANGES_REQUESTED** [review]

**Comment by bitakshat** [comment]
All changes has been done, branch is ready to merge or can be reviewed again.

**Pull request merged**